### PR TITLE
Modified The timer

### PR DESCRIPTION
**Pull request by vr-varad**
took endQuiz in the Quiz component from Core component.
@wingkwong check it out

### CODE REVIEW

File: src/lib/Core.jsx [hunk 1]
```diff
@@ -312,7 +315,7 @@ function Core({
               userAnswer={[...userInput].pop()}
             />
           </div>
-          <div>
+          <div style={{display: 'flex',justifyContent: 'space-between',alignItems:'center'}}>
```
Comment (wingkwong) [inline src/lib/Core.jsx:318]: add a className and add the style in `styles.css` instead of using inline style.

**Review by wingkwong — CHANGES_REQUESTED** [review]
Why not just do in this way?

image
image

Also I'm wondering if the layout could have done better.
image

**Comment by vr-varad** [comment]
I can work on the layout and if there is another way for the Timer let me know @wingkwong

**Comment by vr-varad** [comment]
counter
finalTime
will ir work? @wingkwong

**Comment by wingkwong** [comment]
> I am setting the timer inside quiz because if I u do in core then after finishing the quiz the core component is removed and I can't access the timer from the after the finishing page is rendered.

The core component isn't removed. I guess you've put Timer inside the block when `endQuiz` is false. You can see that I can access the timer after finishing the quiz in the screenshots I provided as I put it outside right.

**Pull request closed without merging**